src/obr/cli.py

```diff
@@ -360,6 +360,12 @@ def query(ctx: click.Context, **kwargs):
     multiple=True,
     help="Path(s) to non-logfile(s) to be also added to the repository.",
 )
+@click.option(
+    "--amend",
+    required=False,
+    multiple=True,
```

Comment:
multiple? 

Also, for completeness sake, maybe add a type hint:
```suggestion
    multiple=True,
    type=str,
```

Reply:
Good catch, I removed the `multiple` flag somewhere, but somehow missed to push it.